src/Sylius/Bundle/PayumBundle/DependencyInjection/Compiler/PayumStoragePaymentAliasPass.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\PayumBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+final class PayumStoragePaymentAliasPass implements CompilerPassInterface
```

Comment:
This `CompilerPass` is needed here because the service id will change depending on the `Payment`'s fqdn.
So it will search for the right service and add an alias to it named `sylius.payum.storage.payment`